### DISCUSSION TITLE
How to see cpu usage for this program?

### DISCUSSION THREAD

**Discussion by kun-codes** [luisbocanegra/plasma-panel-colorizer#96] [Q&A] [answered]
I don't know which program I should look for in htop. I installed it via the AUR Package and I don't see any binary files when I run `yay -Ql plasma6-applets-panel-colorizer`.

**Comment by luisbocanegra** [comment, marked as answer]
Since Plasma add-ons runs inside the Plasma process you wont be able to see specifically how much CPU/memory a specific widget is using, the only way to get a roughly estimation is to compare the usage with and without the widget ~installed~ running.

Edit: the process is `plasmashell`

**Comment by kun-codes** [comment]
Thank you for answering me
